src/pip/_internal/vcs/subversion.py

```diff
@@ -37,7 +54,57 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def vcs_global_options(self):
```

Comment:
I think `get_vcs_global_options()` would be better (e.g. since it can make a subprocess call).